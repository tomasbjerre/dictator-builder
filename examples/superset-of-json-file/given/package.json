{
  "name": "dictator-builder",
  "version": "0.0.7",
  "scripts": {
    "test": "ls test"
  },
  "dependencies": {
    "chalk": "^4.0.0",
    "clear": "^0.1.0",
    "underscore": "^0.10.2"
  },
  "license": "ISC"
}
