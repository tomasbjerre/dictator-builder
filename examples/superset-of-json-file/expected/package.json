{
  "name": "dictator-builder",
  "version": "0.0.7",
  "scripts": {
    "test": "ls test",
    "build": "ls build"
  },
  "dependencies": {
    "chalk": "^4.0.0",
    "clear": "^0.1.0",
    "underscore": "^1.10.2",
    "@types/glob": "^7.1.1"
  },
  "license": "Apache 2",
  "devDependencies": {
    "@babel/core": "^7.9.0",
    "@babel/preset-env": "^7.9.5"
  }
}